tracer/test/test-applications/security/Samples.Security.AspNetCore5/Pages/DataRazorIastPage.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@page
+@model Samples.Security.AspNetCore5.DataRazorIastPageModel
```

Comment:
We need to duplicate. In netcore2, we cant use the one from Samples.AspNetCore5 because the attribute here works differently, needs the order not to be overriden. The model is also not taken into account if it's a shortcut when building.

I followed the same approach as the one followed in the existing DataRazorPage.cshtml